types/transform-pouch/tslint.json

```diff
@@ -0,0 +1,6 @@
+{ "extends": "dtslint/dt.json",
+    "rules": {
+        "no-single-declare-module": false,
+        "no-declare-current-package": false
```

Comment:
Those rules were removed because when implemented, the Plugin exposed by the parent type became unaccessible

Reply:
can you remove those from configuration file and add those inline? (as per README). This would help with readability and maintenance